### PR TITLE
Adds MVP info

### LINKED ISSUE #221: Get round MVP
It seems that [demoinfocs-goland](https://github.com/markus-wa/demoinfocs-golang/blob/master/docs/game-events.md) has a _round_mvp_ event. Is possible to get this information from awpy?

### PR DESCRIPTION
**Pull request by agsalguero**
Fix #221

### CODE REVIEW

File: awpy/types.py [hunk 1]
```diff
@@ -482,6 +482,8 @@ class GameRound(TypedDict):
     weaponFires: Optional[list[WeaponFireAction]]
     flashes: Optional[list[FlashAction]]
     frames: Optional[list[GameFrame]]
+    mvp: Players
```
Comment (JanEricNitschke) [inline awpy/types.py:485]: I dont think the `gameRounds` dict contains a `Players` dict for the key `mvp` from your implementation.

I think it should have an `MVPSteamID: Optional[int]` and the same for MVPName as optional string.

**Review by JanEricNitschke — COMMENTED** [review]
This already looks pretty great.

I just have a couple fairly small comments.

It would also be good if you could add test coverage to the parts you added on the python side.

Like checking the MVP scores in the stats module (download the reference demo and double check what you get with what is shown in the game and on hltv and then add that as a test).

You should also add a small test that the stats function works correctly if the MVPName and/or steamid are `None` or a bot is the mvp.

I think there should already be some examples that you can have a look at in that regard in the stats test file https://github.com/pnxenopoulos/awpy/blob/main/tests/test_stats.py but if you have any questions just ask.

File: awpy/parser/parse_demo.go [hunk 2]
```diff
@@ -129,6 +129,9 @@ type GameRound struct {
 	WinningTeam          *string            `json:"winningTeam"`
 	LosingTeam           *string            `json:"losingTeam"`
 	Reason               string             `json:"roundEndReason"`
+	MVPName              string             `json:"MVPName"`
```
Comment (JanEricNitschke) [inline awpy/parser/parse_demo.go:132]: I think according to the demoinfocs-golang documentation the player can be nil and so we should handle this like we do the other cases where that can happen.

https://pkg.go.dev/github.com/markus-wa/demoinfocs-golang/v3@v3.3.0/pkg/demoinfocs/events#RoundMVPAnnouncement

https://github.com/pnxenopoulos/awpy/blob/main/awpy/parser/parse_demo.go#L205

**Comment by ricardoruwer** [comment]
@agsalguero @JanEricNitschke let's go forward with this PR? 😄

**Comment by JanEricNitschke** [comment]
@ricardoruwer There are still the open points i have mentioned + the PR has to be rebased to the current main. Sadly i dont have the time to do so at the moment.

But feel free to fork their repo and finish the PR from that.

**Comment by agsalguero** [comment]
I'll have some time in a week or two. I'll take a look to it.
